html/index.html.haml

```diff
@@ -1,405 +1,38 @@
-        #unmanaged_files_container.scope{"data-scope" => "unmanaged_files"}
-          %scope-header{ |
-            :summary => "'#{scope_help('unmanaged_files')}'", |
-            :logo => "'assets/logo-unmanaged-files.png'", |
-            :title => "'Unmanaged Files'", |
-            :meta => "description.meta.unmanaged_files", |
-            :singular => "'file'", |
-            :plural => "'files'", |
-            :count => "description.unmanaged_files.files.length" |
-          } |
-          .row.scope_content.collapse.in
-            .col-xs-1
-            .col-xs-11
-              %table.table.table-striped.table-hover.table-condensed.files-table
-                %thead
-                  %tr
-                    %th Name
-                    %th Type
-                %tbody
-                  %tr{"ng-repeat" => "file in description.unmanaged_files.files | filter:query"}
-                    %td{"ng-switch" => true, "on" => "file.downloadable"}
-                      %a.file-download{"ng-switch-when" => "true", "href" => "#"}
-                        {{file.name}}
-                      %span{"ng-switch-default" => true}
-                        {{file.name}}
-                    %td {{file.type}}
-
-    %script{:src => "assets/show/machinery.js"}
-    %script{:src => "assets/bootstrap-tooltip.js"}
-    %script{:src => "assets/bootstrap-popover.js"}
-
-  %body{"data-description" => description_name}
+      = @description.name + " - Machinery System Description"
+    %meta{ :charset => 'utf-8'}/
+    %link{ :href => "assets/machinery-base.css", :rel => "stylesheet", :type => "text/css" }
+    %link{ :href => "assets/machinery.css", :rel => "stylesheet", :type => "text/css" }
+    %script{ :src => "assets/jquery-2.1.1.min.js" }
+    %script{ :src => "assets/transition.js" }
+    %script{ :src => "assets/collapse.js" }
+    %script{ :src => "assets/modal.js" }
+    %script{ :src => "assets/jquery.searcher.min.js" }
+    %script{ :src => "assets/show/machinery.js" }
+    %script{ :src => "assets/bootstrap-tooltip.js" }
+    %script{ :src => "assets/bootstrap-popover.js" }
+
+  %body{ "data-description" => @description.name}
     #file-modal.modal.fade
       .modal-dialog.modal-lg
         .modal-content
           .modal-header
-            %button.close{"type" => "button", "data-dismiss" => "modal"}
+            %button.close{ "type" => "button", "data-dismiss" => "modal" }
               &times;
             %h4#file-modal-title
           .modal-body
-            %textarea#file-modal-file-content{"readonly" => "true"}
-            #file-modal-error{"style" => "display: none"}
+            %textarea#file-modal-file-content{ "readonly" => "true" }
+            #file-modal-error{ "style" => "display: none" }
           .modal-footer
-            %a#file-modal-download-link.btn.btn-success{"target" => "_blank"}
+            %a#file-modal-download-link.btn.btn-success{ "target" => "_blank" }
```

Comment:
Classes should be listed before IDs (.btn should precede #file-modal-download-link)